govcd/user.go

```diff
@@ -85,13 +85,13 @@ func (adminOrg *AdminOrg) FetchUserByHref(href string) (*OrgUser, error) {
 	return orgUser, nil
 }
 
-// FetchUserByName retrieves a user within an admin organization by name
+// GetUserByName retrieves a user within an admin organization by name
 // Returns a valid user if it exists. If it doesn't, returns nil and ErrorEntityNotFound
 // If argument refresh is true, the AdminOrg will be refreshed before searching.
 // This is usually done after creating, modifying, or deleting users.
 // If it is false, it will search within the data already in memory (useful when
 // looping through the users and we know that no changes have occurred in the meantime)
-func (adminOrg *AdminOrg) FetchUserByName(name string, refresh bool) (*OrgUser, error) {
+func (adminOrg *AdminOrg) GetUserByName(name string, refresh bool) (*OrgUser, error) {
```

Comment:
Isn't FetchUserByName was already release and we have to deprecate?

Reply:
Blast! This change was supposed to be in 2.3 already.
But we have changed our collective mind a few times. 
I will change to deprecation + new methods.

Reply:
fixed